### PR TITLE
Contributing guide

### PR DESCRIPTION
**Pull request by paulbakker**
This is the first version - I'm sure we'll come up with more to add in the future.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+This guide helps to find the most efficient way to contribute, ask questions, and report issues.
+
+What is the DGS framework
+----
+
+The DGS framework aims to make GraphQL server development easy, based on Spring Boot.
+It also comes with a code generation plugin to generate Java/Kotlin code from a GraphQL schema.
+
+The DGS framework is written in Kotlin, but primarily targets Java developers. Each feature must work equally well in
+Java and Kotlin. r The framework is based on Spring Boot, and does not target other environments. The framework is
```
Comment (berngp) [inline CONTRIBUTING.md:11]: `. r The` typo

**Review by berngp — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+The DGS framework aims to make GraphQL server development easy, based on Spring Boot.
+It also comes with a code generation plugin to generate Java/Kotlin code from a GraphQL schema.
+
+The DGS framework is written in Kotlin, but primarily targets Java developers. Each feature must work equally well in
+Java and Kotlin. The framework is based on Spring Boot, and does not target other environments. The framework is
+designed to have minimal external dependencies aside from Spring.
+
+I have a question!
+-----
+
```
Comment (srinivasankavitha) [inline CONTRIBUTING.md:16]: We have dedicated => We have a dedicated

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+We have dedicated [discussion forum](https://github.com/Netflix/dgs-framework/discussions) for asking "how to" questions
+and to discuss ideas. The discussion forum is a great place to start if you're considering creating a feature request or
+work on a Pull Request. Please do not create issues to ask questions.
```
Comment (srinivasankavitha) [inline CONTRIBUTING.md:19]: Bold "Please do not create issues to ask questions"

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+I have a great idea for a new feature
+----
+Many features in the framework have come from ideas from the community. If you think something is missing or certain use
+cases could be supported better, let us know!
+You can do so by opening a discussion on the [discussion forum](https://github.com/Netflix/dgs-framework/discussions).
+Provide as much relevant context to why and when the feature would be helpful. Providing context is especially important
+for "Support XYZ" issues since we might not be familiar with what "XYZ" is and why it's useful. If you have an idea of
+how to implement the feature, include that as well.
+
```
Comment (srinivasankavitha) [inline CONTRIBUTING.md:70]: Once we decided on a direction => Once we have decided on a direction

It's time to summarize the idea in an issue => It's time to summarize the idea by creating a new issue.

**Review by srinivasankavitha — CHANGES_REQUESTED** [review]

**Pull request merged**